From e522e342c58994c3fc252cc78f55acc2e93d28be Mon Sep 17 00:00:00 2001
From: Tomas Henzl <thenzl@redhat.com>
Date: Mon, 8 Sep 2014 14:31:32 -0400
Subject: [scsi] hpsa: add HP/3PAR vendor id to pci_ids.h

Message-id: <1410186734-19986-42-git-send-email-thenzl@redhat.com>
Patchwork-id: 92053
O-Subject: [RHEL6.6 e-stor PATCH 41/83] hpsa: add HP/3PAR vendor id to pci_ids.h
Bugzilla: 1069185
RH-Acked-by: Maurizio Lombardi <mlombard@redhat.com>
RH-Acked-by: David Milburn <dmilburn@redhat.com>
RH-Acked-by: Ewan Milne <emilne@redhat.com>

Signed-off-by: Stephen M. Cameron <scameron@beardog.cce.hp.com>
Signed-off-by: James Bottomley <JBottomley@Parallels.com>
(cherry picked from commit 4b409c50343f41596f76006460cb5adc4d25824e)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/include/linux/pci_ids.h b/include/linux/pci_ids.h
index 8b77761..d7786b2 100644
--- a/include/linux/pci_ids.h
+++ b/include/linux/pci_ids.h
@@ -726,6 +726,7 @@
 #define PCI_DEVICE_ID_SI_7018  0x7018
 
 #define PCI_VENDOR_ID_HP  0x103c
+#define PCI_VENDOR_ID_HP_3PAR  0x1590
 #define PCI_DEVICE_ID_HP_VISUALIZE_EG 0x1005
 #define PCI_DEVICE_ID_HP_VISUALIZE_FX6 0x1006
 #define PCI_DEVICE_ID_HP_VISUALIZE_FX4 0x1008
-- 
1.7.1